themes/src/main/resources/theme/base/login/resources/js/webauthnAuthenticate.js

```diff
@@ -91,6 +91,9 @@ export function returnSuccess(result) {
     if (result.response.userHandle) {
         document.getElementById("userHandle").value = base64url.stringify(new Uint8Array(result.response.userHandle), { pad: false });
     }
+    if (result.response.platformAttachment) {
```

Comment:
If I'm not wrong this should be [result.authenticatorAttachment](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential/authenticatorAttachment) which can have the values `platform` or `cross-platform`. Or am I missing something?